### DISCUSSION TITLE
Suggestion about interface

### DISCUSSION THREAD

**Discussion by maximka76667** [gradejs/gradejs#37] [Ideas]
Hi! 
Your project seems very interesting and it works fine, I just wanted to make a small suggestion. 
When I entered your page for the first time and tried to use it I was a little bit confused that I need to return to the main page to analyze another website. I think it could be better to have some form on the "result" page to look for another page.
![Screenshot 2022-06-14 204837](https://user-images.githubusercontent.com/74974283/173666519-a8e08480-0fbb-4d2a-97f6-7fed85ba102d.png)

**Comment by spalt08** [comment]
Hi @maximka76667 
Thanks for the suggestion! We are currently working on the new design mockups and this tip is great.

**Comment by spalt08** [comment]
@maximka76667 
We've released the redesigned version
